### DISCUSSION TITLE
A general Thank You, and Anything New?

### DISCUSSION THREAD

**Discussion by SirNibble** [prirun/p50em#30] [General]
Blimey, those heady days of March 2021 feel a life time ago now.

Discovering the Pi version of the emulator was such a god-send during the last UK lockdown, and you've really just no idea how much it helped me get through it. Kicking the old 'muscle-memory' back to life was such an exhilarating experience ... with a side-order of "Good Grief ... I'm old!" :)

Now I'm back to work full time I've really no-longer the time, or the energy, to use my dedicated emulated system (PIP01) as much as I would like. But every now and then, I'll login, fire up a version of LIFE I did in Pascal using a VT100 ESCape code 'library' I re-wrote from 80s memory, and just have it pootling away in the corner of my desktop.

So, I just wanted to say a proper Thank You to Jim and Dennis for making this old PriMate's life just that much brighter during what's been a pretty damn dark time.

Thank You.

Now ... what's next? 

Was the Modula 2 compiler ever fixed?

Lord help us .... PRIMIX working on the emulator? Gotta be better than the real thing .... we tried it a few times on our 9955ii and vowed never to use it again! :)

**Comment by drboone** [comment]
Glad it gave you some useful distraction, I'm sure we all needed that.

I haven't solved the MODULA compiler loop problem yet.  I think the PASCAL problem is sorted out.  We concluded that a Rev24 install problem that cropped up is actually a PRIMOS bug.

Would be nice to have PRIMIX, even if it was a dog, but as far as I know it's not in captivity.  I _am_ getting > 100 MIPS on my home desktop these days, so it wouldn't be _all_ that awful. :)

There is the Georgia Tech version of Software Tools, if you wan't to fiddle with that.
